Store/admin/components/Account/Details.php

```diff
@@ -115,12 +115,13 @@ protected function getTableModel(SwatView $view)
 		case 'payment_methods_view':
 			return $this->getPaymentMethodsTableModel($view);
 		}
+		return null;
 	}
 
 	// }}}
 	// {{{ protected function getOrdersTableModel()
 
-	protected function getOrdersTableModel(SwatTableView $view)
+	protected function getOrdersTableModel(SwatTableView $view): SwatDBDefaultRecordsetWrapper
```

Comment:
Using the more generic type SwatTableModel makes sense here.

Reply:
Except in this method we absolutely _know_ that it's going to return a `SwatDBDefaultRecordsetWrapper`.  So why not show that as the return type?

I know this method is basically a "sub-method" of `getTableModel` above, (which extends the more generically-typed `getTableModel(): ?SwatTableModel`).  But maybe there are cases where the sub-methods are used elsewhere in the code, so having them be as narrowly typed as possible makes more sense to me.